eosc/cmd/systemNewaccount.go

```diff
@@ -44,6 +44,8 @@ active:
     weight: 1
 ---
 
+For use with any chain other than the EOS mainnet, add the flag `--contract [contract_name]`
```

Comment:
`--contract [contract_name]` Feels not clear, is it the system contract, you should probably make it clearer and show two examples for EOS and BOS maybe.